migration-tests/utils/sovereign_account_translation.ts

```diff
@@ -0,0 +1,543 @@
+
+// Account translation maps for sovereign accounts and their derived accounts.
+import { hexToU8a } from '@polkadot/util';
+
+export interface TranslationEntry {
+  rcAccount: Uint8Array;
+  rcAddress: string;
+  ahAccount: Uint8Array;
+  ahAddress: string;
+}
+
+export interface DerivedTranslationEntry {
+  rcAccount: Uint8Array;
+  rcAddress: string;
+  derivationIndex: number;
+  ahAccount: Uint8Array;
+  ahAddress: string;
+}
+
+// List of RC para to AH sibl sovereign account translation.
+// Note: This data will be sorted by rcAccount hex values for binary search optimization.
+const SOV_TRANSLATIONS_RAW: TranslationEntry[] = [
+  // para 0
+  {
+    rcAccount: hexToU8a("7061726100000000000000000000000000000000000000000000000000000000"),
+    rcAddress: "5Ec4AhNmFmpEvbFUqLoWAd68CcvudkegML6YhBEg1keWpPaf",
+    ahAccount: hexToU8a("7369626c00000000000000000000000000000000000000000000000000000000"),
+    ahAddress: "5Eg2fnsaLW194BydPLkbHGRwT6cDMAhnQyjyx8mAtUg5g6ei",
+  },
+  // para 2048
+  {
+    rcAccount: hexToU8a("7061726100080000000000000000000000000000000000000000000000000000"),
+    rcAddress: "5Ec4AhNmG8sxwXqncL8VfyMnPftV1mJcwwpdokxfDhZ91zK2",
+    ahAccount: hexToU8a("7369626c00080000000000000000000000000000000000000000000000000000"),
+    ahAddress: "5Eg2fnsaLs4s58ZwAL5anchbe9ZnjBMj1bU54iVA6Rahsb2S",
+  },
+  // para 2050
+  {
+    rcAccount: hexToU8a("7061726102080000000000000000000000000000000000000000000000000000"),
+    rcAddress: "5Ec4AhNmfNzKymhQFb5x4LdvawDKJVAs9NGH4yHnQHd5U6Af",
+    ahAccount: hexToU8a("7369626c02080000000000000000000000000000000000000000000000000000"),
+    ahAddress: "5Eg2fnsak7BE7NRYob33AyyjqQtd1uDyD1uiKvpHH1eeKrjD",
+  },
+  // para 2051
+  {
+    rcAccount: hexToU8a("7061726103080000000000000000000000000000000000000000000000000000"),
+    rcAddress: "5Ec4AhNms13WVtdD5DZgFXGzgZsjSr6zF5V6haTLzaf3h7Wo",
+    ahAccount: hexToU8a("7369626c03080000000000000000000000000000000000000000000000000000"),
+    ahAddress: "5Eg2fnsawjEQdVMMdDWmNAcow3Z3AGA6Jj8XxXyqsJgcZ15F",
+  },
+  // para 3334
```

Comment:
Please also make a secondary list with the exact values from the GDoc. We should not rely on the output of my script for both tests.  
This list here can stay but i want to 100% make sure that the addresses from the Gdoc are tested explicitly.